src/milia/utils/file.clj

```diff
@@ -2,6 +2,8 @@
   (:require [clojure.java.io :as io])
   (:import [org.apache.commons.io IOUtils]))
 
+(def file-extension-regex #"\w+$")
```

Comment:
This regex pattern seems to match any word. Which file extension were we targeting?

Reply:
We are just targeting json file extensions.

Reply:
Let me check if i can one that is specific to this.

Reply:
Will use this instead `#".json$"`